.github/workflows/validation.yaml

```diff
@@ -0,0 +1,52 @@
+name: Validation
+on:
+  push:
+    branches:
+      - '*'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}
+  cancel-in-progress: true
```

Comment:
This cancels the running job if a new one comes up?

Reply:
This will cancel the running jobs if a new commit pushed to the same branch.